docs/conf.py

```diff
@@ -318,6 +317,44 @@ def run(self):
             sys.stdout = old_stdout
 
 
+class MiniGallery(Directive):
+    """
+    Custom directive to insert a mini-gallery
+
+    The required argument is the name of the object, excluding the initial `sunpy.`.  There can be
```

Comment:
I get why you have dropped the initial `sunpy.` but I think it would be better if we kept it. For a start affiliated packages could import this from core and use it in their docs which would obviously not work so good :D

Reply:
Okay, changed